ads/README.md

```diff
@@ -0,0 +1,94 @@
+# Integrating ad networks into AMP
+
+See also our [ad integration guidelines](../3p/README.md#ads).
+
+## Overview
+Ads are just another external resource and must play within the same constraints placed on all resources in AMP. We aim to support a large subset of existing ads with little or no changes to how the integrations work. Our long term goal is to further improve the impact of ads on the user experience through changes across the entire vertical client side stack.
+
+
+## Constraints
+A summary of constraints placed on external resources such as ads in AMP HTML:
+
+- Because AMPs are served on HTTPS and ads cannot be proxied ads must be served on HTTPS.
+- The size of an ad unit must be static. It must be knowable without fetching the ad and it cannot change at runtime except through https://github.com/ampproject/amphtml/issues/728.
```

Comment:
Suggest changing text to: "except through iframe resizing" (with link to issue), and add a note in that issue to update this doc